root/statistics/StatisticsLayout.js

```diff
@@ -33,7 +32,7 @@ type TabPropsT = {
 
 const LinkStatisticsTab = ({link, title, page, selected}: TabPropsT) => (
   <li className={page === selected ? 'sel' : ''}>
-    <a href={link}>{title}</a>
+    <a href={link}>{l(title)}</a>
```

Comment:
This seems broken, since `title` isn't guaranteed to be in English due to the `l()` calls in `infoLinks` returning an undefined language.

The correct fix is to have the `title` props in `infoLinks` use `N_l`.

Reply:
This doesn't seem to be resolved (sorry if I'm missing something).

Reply:
Should be resolved by https://github.com/metabrainz/musicbrainz-server/pull/964/commits/968169d3e2a177e9a91f69e5196a39dbaaee9765 and https://github.com/metabrainz/musicbrainz-server/pull/964/commits/9dcd282f702943d0461d1907032affca477ed1af now.

Reply:
... did I forget to push again? Sigh. Well I guess it doesn't matter if @yvanzo already did it.

Reply:
(I'd rather you let me fix my own mess myself next time though if that's ok!)

Reply:
(Sure, just wanted to help with a PR we all focused on, didn’t notice you were online too atm.)